rc_user

```diff
@@ -2,3 +2,4 @@ export USERNAME='$(shell whoami)'
 export API_URL=//mf-chsdi3.dev.bgdi.ch/$USERNAME
 export APACHE_BASE_PATH=/$USERNAME
 export E2E_TARGETURL=https://mf-geoadmin3.dev.bgdi.ch/$USERNAME
+export SHOP_URL=//shop-shopbgdi.openshift.puzzle.ch
```

Comment:
We need this url in rc_dev, rc_int and rc_prod (to the corresponding shop urls).